### DISCUSSION TITLE
Need help setting up custom Tiger file

### DISCUSSION THREAD

**Discussion by marlin-mixon** [mediagis/nominatim-docker#576] [Q&A]
Having issues trying to establish a custom Tiger file for Texas.  I'm probably trying to make it too complicated.  

My custom file is in ~/nominatim/texas-tiger-preprocessed.csv.tar.gz

I created a Dockerfile file that puts ~/nominatim/texas-tiger-preprocessed.csv.tar.gz into /app/tiger/texas-tiger-preprocessed.csv.tar.gz
(any easier way than setting up a Dockerfile?)

my RUN command:
sudo docker run -it \
  -e PBF_URL=https://download.geofabrik.de/north-america/us/texas-latest.osm.pbf \
  -e REPLICATION_URL=https://download.geofabrik.de/north-america/us/texas-updates/ \
  -e IMPORT_US_POSTCODES=true \
  -e IMPORT_TIGER_ADDRESSES=/app/texas-tiger-preprocessed.csv.tar.gz \
  -e IMPORT_STYLE=full \
  -e NOMINATIM_PASSWORD=xxxxxxxxx \
  -p 8080:8080 \
  -p 5432:5432 \
  --name nominatim \
  -v nominatim-data:/var/lib/postgresql/14/main \
  nomghh

Ultimately it fails here:
2024-08-18 18:40:20: Using project directory: /nominatim
2024-08-18 18:40:26: Creating database
2024-08-18 18:40:29: Setting up country tables
2024-08-18 18:40:36: Importing OSM data file
2024-08-18 18:40:36: Configuration file '__IMPORT_STYLE__' not found.
Directories searched: [PosixPath('/nominatim'), PosixPath('/usr/local/etc/nominatim')]

**Comment by mtmail** [comment]
`Configuration file 'IMPORT_STYLE' not found.` looks more like the docker scripts added an error to the `.env` file. I think `init.sh` writes to the file. If possible inspect the file `/nominatim/.env` inside the container, maybe it's something obvious.

TIGER data can be added after an import, as a separate step, it doesn't have to be part of the original import (unlike wikipedia data). 


I'd try that https://nominatim.org/release-docs/4.4/customize/Tiger/ Outside the Docker container the command would be
```bash
docker exec -it nominatim sudo -u nominatim nominatim add-data --tiger-data ...
```
in your case I think
```bash
docker exec -it nomghh sudo -u nominatim nominatim add-data --tiger-data /app/texas-tiger-preprocessed.csv.tar.gz
```
